ec2uploadimg

```diff
@@ -413,7 +427,7 @@ try:
                                                    'ami',
                                                    '--ec2-ami')
                 except Exception:
-                    print('Could not determine helper AMI-ID', file=sys.stderr)
+                    print_ex(args.verbose, 'Could not determine helper AMI-ID')
```

Comment:
> Not sure I'm following your comment correctly though.

I was just talking about that case, where we only output a message, without printing the message from the actual exception.